.github/workflows/makecommit.yml

```
@@ -0,0 +1,21 @@
+name: Make Generate
+on: push
+
+jobs: 
+  run:
+    name: Make Generate Template
+    runs-on: ubuntu-latest
+    steps: 
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Make
+      run: make clean && make
```

```suggestion
      run: make clean generate-syncset
```
This replicates the default target but it's explicit.